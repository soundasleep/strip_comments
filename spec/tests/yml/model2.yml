model:
  a:
    string: it's a string
  b:
    string: # ignored
  c:
    string: it's another string