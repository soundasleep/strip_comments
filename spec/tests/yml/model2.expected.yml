model:
  a:
    string: it's a string
  b:
    string: 
  c:
    string: it's another string